tests/test_remote_client.py

```diff
@@ -75,7 +75,6 @@ def test_get_fs_info(self):
         fs_item_id = FS_ITEM_ID_PREFIX + "fakeId"
         with pytest.raises(NotFound):
             remote.get_fs_info(fs_item_id)
-        assert not remote.get_fs_info(fs_item_id, raise_if_missing=False)
```

Comment:
Why this deletion?

Reply:
We removed the `raise_if_missing` argument from the `get_fs_info()` function. Now, if we call it, it will always raise a `NotFound`, which is already tested in the instruction just above it.